cmd/avrogen/main.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	g "github.com/hamba/avro/gen"
+)
+
+type rawOpts struct {
+	Package string
+	OutFile string
+	Schema  string
+	Tags    string
+	Help    bool
+}
+
+func main() {
+	var ro rawOpts
+	flag.StringVar(&ro.Package, "pkg", "", "-pkg <file-name>")
+	flag.StringVar(&ro.OutFile, "o", "", "-o <file-name>")
+	flag.StringVar(&ro.Schema, "schema", "", "-schema <file-name>")
+	flag.BoolVar(&ro.Help, "h", false, "-h")
+	flag.Parse()
```

Comment:
On CLI's I usually load the version in through the `ldflags` and have the CI manage what gets injected. Questions:
1. Do you want a `-version` option?
1.2 If so, do you want it as a `const` in code, `ldflags` or some other way